### ISSUE TITLE
Reset button for each Theme

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1661]
i am suggesting to have in administrator --> themes --> [themename]  next the Save & Cancel button the new one with name "Reset to default" or something similary.

Many people are trying to change some colors, but after they are lost to change back the theme to default view as on beginning :) 

Please let me know your opinions :)

![template](https://cloud.githubusercontent.com/assets/5773777/16413396/f40d5070-3d31-11e6-8d25-ebc635f2c8f5.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Reset button for each Theme](https://trello.com/c/987O2mRL/92-reset-button-for-each-theme)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
